src/server/components/features/features-list/EnableChartEditor.ts

```diff
@@ -4,7 +4,7 @@ import {createFeatureConfig} from '../utils';
 export default createFeatureConfig({
     name: Feature.EnableChartEditor,
     state: {
-        development: false,
+        development: true,
```

Comment:
This flag can enable `save as editor script` button for wizard. Do you really need them?